src/plugins/discover/public/application/main/hooks/use_discover_state.ts

```diff
@@ -169,7 +167,7 @@ export function useDiscoverState({
     documents$: data$.documents$,
     dataViews,
     stateContainer,
-    dataViewList: savedDataViewList,
+    dataViewList: initialDataViewList,
```

Comment:
even simpler `dataViewList` was mapped to `savedDataViewList` and then it was assigned to dataViewList ... well, now it's just `dataViewList` and it will live happily ever after ... or until this while thingy will soon go away ... in a follow up, we this part will be migrated to be part of the state container

Reply:
thx!

Reply:
done